frontend/src/components/ConfigBar/ConfigBar.vue

```diff
@@ -84,10 +82,17 @@ export default {
     },
     //Update tabsData[i].isActive props if tab changes
     onTabOneClick() {
-      this.$emit("tabOneActive")
+      //Close all parameter descriptions for current tab
+      for (let i = 0; i < this.$refs.item.length; i++) {
+        this.$refs.item[i].tabClick()
+      }
```

Comment:
```suggestion
      this.$refs.item.forEach((item) => item.tabClick())
      
      // OR
      
      this.$refs.item.forEach((item, index) => this.$refs.item[i].tabClick())
```

You should try to use ES6 arrow functions where you can, more clean and generally the standard. (One of the above will work, ideally the top but bottom may only work)

Reply:
if item is an array []